src/views/projects/SourceBrowser/FileDiff.svelte

```diff
@@ -193,19 +216,62 @@
       {#if file.diff.type === "plain"}
         {#if file.diff.hunks.length > 0}
           <table class="diff">
-            {#each file.diff.hunks as hunk}
+            {#each file.diff.hunks as hunk, hunkIdx}
               <tr class="diff-line">
                 <td colspan={2} />
                 <td colspan={6} class="diff-expand-header">
                   {hunk.header}
                 </td>
               </tr>
-              {#each hunk.lines as line}
-                <tr class="diff-line" data-expanded data-type={lineSign(line)}>
-                  <td class="diff-line-number left" data-type={lineSign(line)}>
+              {#each hunk.lines as line, lineIdx}
+                {@const hunkLineSelected =
+                  selectedHunkLine == "H" + hunkIdx + "L" + lineIdx}
+                {#if hunkLineSelected}
+                  {#if !initialScroll}
+                    {(setTimeout(function () {
+                      const target = document.getElementById(
+                        file.path + "::H" + hunkIdx + "L" + lineIdx,
```

Comment:
- scroll now works, regardless of two stage loading, as it's initiated here manually 
- notice, target id is actually `::H...` whereas hash would be `:H`, that's because I couldn't `e.preventDefault()` or `e.stopPropagation()` inside `updateSelectedLineId`, so now initial scroll would work which is desired behavior, but any further click on lines would force scroll, which is undesired